CalibMuon/DTCalibration/python/PlottingTools/plotResiduals.py

```diff
@@ -2,12 +2,22 @@
 from .fitResidual import fitResidual
 from .drawHistoAllChambers import drawHisto
 
-def plot(fileName,sl,dir='DQMData/Run 1/DT/Run summary/DTCalibValidation',option="HISTOPE1",draw=True):
+# This script is not to be run standalone.
+# Use CalibMuon/DTCalibration/scripts/allPlotsResiduals.py instead.
 
-    mean_ymin = -0.02
-    mean_ymax =  0.02
-    sig_ymin = 0.
-    sig_ymax = 0.07
+def plot(fileName,sl,dir='DTResiduals', run='1',option="HISTOPE1",draw=True):
+
+    if run in ['390106','390170']:
```

Comment:
Are you sure that these are the only cosmics runs around?

Reply:
Of course these are not the only cosmic runs. But we only run the workflow on a few cosmics runs, and it is enough to have them hardcoded here. This is only for changing ranges in the plots, so please let it pass.